### PR TITLE
pep8 upgrade and fixes

### PR DESCRIPTION
**Pull request by setrofim**

### CODE REVIEW

File: dev_scripts/README [hunk 1]
```diff
@@ -15,8 +15,8 @@ Scripts
 :get_apk_versions: Prints out a table of APKs and their versons found under the
                    path specified as the argument.
 
-:pep8: Runs pep8 code checker (must be installed) over wlauto with the correct 
-       settings for WA.
+:pep8: Runs pyclodestyle (formerly called "pep8") code checker (must be
```
Comment (Sticklyman1936) [inline dev_scripts/README]: Shouldn't this be flake8?

File: dev_scripts/README [hunk 2]
```diff
@@ -15,8 +15,8 @@ Scripts
 :get_apk_versions: Prints out a table of APKs and their versons found under the
                    path specified as the argument.
 
-:pep8: Runs pep8 code checker (must be installed) over wlauto with the correct 
-       settings for WA.
+:pep8: Runs flake8 (formerly called "pep8") code checker (must be
+       installed) over wlauto with the correct settings for WA.
```
Comment (marcbonnici) [inline dev_scripts/README]: wlauto -> wa

File: wa/framework/workload.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#    Copyright 2014-2018 ARM Limited
+#    Copyrighe 2014-2018 ARM Limited
```
Comment (marcbonnici) [inline wa/framework/workload.py]: Copyrighe -> Copyright

**Comment by setrofim** [comment]
Comments addressed.

**Pull request merged**